providers/k8s/connection/admission/connection.go

```diff
@@ -66,6 +66,10 @@ func (c *Connection) SupportedResourceTypes() (*resources.ApiResourceIndex, erro
 	return c.ManifestParser.SupportedResourceTypes()
 }
 
+func (c *Connection) SetID(id uint32) {
```

Comment:
I am not sure we want to let someone outside of the provider overwrite the connection id

Reply:
It's gone now